cypress.json

```diff
@@ -1,4 +1,5 @@
 {
+  "projectId": "ch3eek",
```

Comment:
Will we have access to the project on Cypress since they're hosting the videos? What I'd you're out of town and run out of space

Reply:
Going to send you invite link ser